### DISCUSSION TITLE
how can i use model viewer still in dev?

### DISCUSSION THREAD

**Discussion by maorp** [google/model-viewer#1797] [General]
Hi, how can i use model viewer still in dev? (meaning unreleased) and use it as:  
<script type="module" src="../../node_modules/@google/model-viewer/dist/model-viewer.js"> </script>
i mean without building modelviewer myself... maybe there is unpkg link somewhere to the latest build?
thanks :)

**Comment by elalish** [comment]
You can test with the one we serve on our dev pages, which is master. Just don't use it for production! `<script type="module" src="https://modelviewer.dev/node_modules/@google/model-viewer/dist/model-viewer.js"> </script>`
